cpu/lpc2387/cpu.c

```diff
@@ -110,4 +113,22 @@ void arm_reset(void)
     while(1) {}
 }
 
+/**
+ * @brief Initialize the CPU, set IRQ priorities, clocks
+ */
+void cpu_init(void)
+{
+    /* configure CPU clock */
+    cpu_init_clks();
+
+    /* set up GPIOs */
+    gpio_init_ports();
+
```

Comment:
I think `board_init()` should be best called here, as it sets up UART0 needed for stdio.

Reply:
Ok.